sql/CMakeLists.txt

```diff
@@ -120,7 +120,7 @@ DTRACE_INSTRUMENT(sql)
 TARGET_LINK_LIBRARIES(sql ${MYSQLD_STATIC_PLUGIN_LIBS} 
   mysys mysys_ssl dbug strings vio pcre ${LIBJEMALLOC}
   ${LIBWRAP} ${LIBCRYPT} ${LIBDL} ${CMAKE_THREAD_LIBS_INIT}
-  ${SSL_LIBRARIES})
+  ${LIBSYSTEMD} ${SSL_LIBRARIES})
```

Comment:
I'm not sure I like the run-time dependency on libsystemd-daemon.
I've just checked on trusty that uses systemd.
Three services are socket activated. None depends on libsystemd-daemon.
ssh and acpid use StandardInput=socket
dbus uses sd_listen_fds, but doesn't depend on libsystemd-daemon,
dunno why.

Apparently, we cannot use StandardInput=socket
But we cannot add a new rpm/deb dependency in 10.0
(I've tried that with jemalloc).

Options?
* static libsystemd-daemon does not exist, as far as I can see. pity,
  that's what we used with jemalloc
* include it into the source tree? and remove in 10.1?
* use a limited systemd support in 10.0 and only add
  notify/listen in 10.1?
* add full support, but don't compile it in (leave it to users who
  self-compile and to distributions)?
* something else?